lib/origen/application/runner.rb

```diff
@@ -134,30 +134,40 @@ def prepare_and_validate_workspace(options = {})
       #
       # @api private
       def record_invocation(options)
-        # record_invocation = false
-        # begin
-        #  # Only record user invocations at this time, also bypass windows since it seems
-        #  # that threads can't be trusted not to block
-        #  unless Origen.running_remotely? # || Origen.running_on_windows?
-        #    record_invocation = Thread.new do
-        #      Origen.client.record_invocation(options[:action]) if options[:action]
-        #    end
-        #  end
-        # rescue
-        #  # Don't allow this to kill an origen command
-        # end
+        if Origen.site_config.record_invocation == true
```

Comment:
I assume this just returns `nil` today for any attribute not mentioned in the config?

Reply:
correct!